compose.yml

```diff
@@ -0,0 +1,23 @@
+services:
+  node_exporter:
+    image: quay.io/prometheus/node-exporter:v1.8.2
+    container_name: node_exporter
+    command: 
+      - '--path.rootfs=/host'
+    network_mode: host
+    pid: host
+    restart: unless-stopped
+    volumes:
+      - '/:/host:ro,rslave'
+  dcgm_exporter:
```

Comment:
Do you want to have `restart: unless-stopped` here?

Reply:
do you have opinions on having this otherwise?

Reply:
`dcgm_exporter` is missing `restart: unless-stopped`.  I feel that it should be added, otherwise we risk losing monitoring.

I could further be convinced to switch to `restart: always`.